events/code_review_events/workflow.py

```diff
@@ -167,7 +170,10 @@ def is_blacklisted(self, revision: dict):
         return True
 
     def publish_results(self, payload):
-        assert self.publish is True, "Publication disabled"
+        if not self.publish:
```

Comment:
Nit: we could just skip the `self.api.update_build_target` calls, so even on testing we run all the code except the upload.

Reply:
It should be implemented in libmozdata : https://github.com/mozilla/libmozdata/issues/175